aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,41 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## April 11, 2022 - June 3, 2022
+
+### Communications
+
+- We're looking to improve the usability of the resources we're delivering, making the site easier to search, providing more and better examples, etc. We've got a summer intern coming on board soon to help with this.
+
+### Stay Current with the Latest Changes
+
+- **Page Header**
+  - Both [Figma](https://www.figma.com/file/BHp0KrOcPvbnzNyshfHkd8/HPE-Page-Header?node-id=734%3A40546) and [grommet](https://v2.grommet.io/pageheader) components are now available as well as [design guidance](https://design-system.hpe.design/components/pageheader)
+
+### Roadmap Updates
+
+- **Figma Component Refactoring**
+  - We are currently working on input components. Thanks for all of the help we've received from folks across HPE in this effort.
+- **Multi-Select**
+  - Wrapping up basic anatomy and focusing on how to represent the current selection.
+- **Adoption Scorecard**
+  - Working on initial draft version of a spreadsheet to evaluate how well products and services are aligned to the HPE Design System. Look for more to come in this area soon.
```

Comment:
what about Tabs since I have been out of the loop Im not sure where we are with that but I thought the design was done?

Reply:
I didn't include it because Tabs hasn't been a "roadmap item": https://github.com/orgs/grommet/projects/11
But, willing to add if you think it's notable enough.

Reply:
gotcha some reason I thought it was a big item ticket that's fine to leave off